packages/lesswrong/components/notifications/NotificationsPopover.tsx

```diff
@@ -116,19 +128,25 @@ const NotificationsPopover = ({karmaChanges, markAllAsRead, closePopover, classe
     closePopover?.();
   }, [closePopover, closeMenu]);
 
-  const notifs = useRef<NotificationDisplay[]>([]);
-  if (
-    data?.NotificationDisplays?.results?.length &&
-    data.NotificationDisplays.results.length !== notifs.current.length
-  ) {
-    notifs.current = data.NotificationDisplays.results ?? [];
-  }
```

Comment:
I'm getting a ~bug where a after you click on an unread notification, it appears as unread for a split second when you next open the popover. I wonder if it's related to removing this.

Either way I think this is borderline a feature anyway, because it reminds you of the last thing you clicked on